draft-ietf-quic-transport.md

```diff
@@ -3309,103 +3291,74 @@ ACK Delay:
 
 ACK Block Count:
 
-: A variable-length integer specifying the number of Additional ACK Block (and
-  Gap) fields after the First ACK Block.
+: A variable-length integer specifying the number of ACK Blocks in addition to
+  the first.  A value of 0 indicates that there is only one ACK Block.
 
 ACK Blocks:
 
-: Contains one or more blocks of packet numbers which have been successfully
-  received, see {{ack-block-section}}.
+: An ACK Block is encoded as a variable-length integer.  The encoding of the ACK
+  Block is described more in {{ack-blocks}}.
 
 
-### ACK Block Section {#ack-block-section}
+### ACK Blocks {#ack-blocks}
 
-The ACK Block Section consists of alternating Gap and ACK Block fields in
-descending packet number order.  A First Ack Block field is followed by a
-variable number of alternating Gap and Additional ACK Blocks.  The number of Gap
-and Additional ACK Block fields is determined by the ACK Block Count field.
+Each ACK Block in an ACK frame describes a contiguous range of packets, as well
+as the observed type of the packets in that range.  Each range of packets is
```

Comment:
I think of packet types as Initial/Retry/etc.  How about ack state?  Or something else that doesn't collide with the existing packet type?

Also, "Each range of packets is attributed a type," overlaps with the first sentence.  How about "Each packet range's type is encoded into the two least..."